src/transport/MessageCounter.h

```diff
@@ -106,8 +107,14 @@ class GlobalEncryptedMessageCounter : public MessageCounter
 class LocalSessionMessageCounter : public MessageCounter
 {
 public:
-    static constexpr uint32_t kInitialValue = 1;
-    LocalSessionMessageCounter() : value(kInitialValue) {}
```

Comment:
Did you remove the constructor intentionally? Is the value initialized to `kInitialValue` somewhere else?